static/scss/partials/main.scss

```diff
@@ -1080,7 +1080,7 @@ input:focus::placeholder {
 
 .relay-email-address-label:focus,
 input.input-has-error {
-  width: 200px !important;
+ // width: 200px !important;
```

Comment:
nit: let's go ahead and remove the line rather than comment.

Reply:
Done!